dat/missions/neutral/cargo.lua

```diff
@@ -105,6 +104,9 @@ function accept()
       tk.msg(full[1], full[2]:format(amount, amount - player.pilot():cargoFree()))
       misn.finish()
    end
+   misn.createLog(buildCargoMissionDescription( nil, amount, cargo, destplanet, destsys ),"Trade",0)
+   origin_p, origin_s = planet.cur()
+   misn.appendLog("Cargo accepted on "..origin_p:name() .. " for " .. destplanet:name())
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.